### ISSUE TITLE
Where can I find benchmark results?

### ISSUE THREAD

**Issue by marcinz** [cupy/cupy-performance#7]
I am wondering if the benchmarks in this repo (or some other cupy benchmarks) are run regularly somewhere. If they are, is there a place where I can find the results?

**Comment by emcastillo** [comment]
They aren't 😄

**Comment by kmaehashi** [comment]
This is in our TODO list for a long time... How can we make it public? I'm thinking of:

* For every release, run a benchmark against the wheel. Persist the raw benchmark results and rendered HTML to GCS.
* For every pull-requests, run the benchmark against the code, then compare with the raw benchmark results in GCS, then persist rendered HTML to GCS.

**Comment by leofang** [comment]
@kmaehashi FYI: https://github.com/spcl/npbench

**Comment by jakirkham** [comment]
Interesting hadn't heard of that one before.

Another popular one is [airspeed velocity (asv)]( https://asv.readthedocs.io/en/stable/index.html ), which NumPy & SciPy both use to run and show benchmarks. Here's [NumPy's benchmark results]( https://pv.github.io/numpy-bench/ ).

**Comment by jakirkham** [comment]
cc @pentschev (in case you have thoughts here)

**Comment by pentschev** [comment]
I agree ASV is potentially the best candidate for a library like CuPy as it will provide a historic view on a per-commit basis, making it really useful to catch regressions too.
